lib/managers/KalturaFetchManager.js

```diff
@@ -144,14 +145,15 @@ class KalturaFetchManager extends kaltura.KalturaManager
 				let size = KalturaUtils.getFileSizeInBytes(originalFillerLocalPath);
 				if (!size)
 					rejectCallback(`Fail to download filler from ${downloadUrl} to path ${originalFillerLocalPath} with size of ${size}`);
-				TranscodingHelper.transcodeExistingFileToDisk(flavorId, This.apiConnector, partnerId, originalFillerLocalPath, This.fillerPath).then(
+				TranscodingHelper.transcodeExistingFileToDisk(flavorId, This.apiConnector, partnerId, originalFillerLocalPath, This.fillerPath, null, false).then(
```

Comment:
The default function call to  transcodeExistingFileToDisk will insert null, false 
So you can call the function without those parameters